content/troubleshooting/common-ios-issues.md

```diff
@@ -34,11 +34,14 @@ For example, you're using a **development** certificate with a **distribution**
 Make sure the certificate and the provisioning profile types match.
 
 ###### Missing entitlements in the Apple Developer portal
-If you are missing the required entitlements for your app in the Apple Developer portal, you may see error messages such as:
+In case of getting errors about an existing provisioning profile's missing entitlements, one can be added or updated by finding and editing the profile in **Apple Developer portal > Certificates, identifier & profiles > Identifiers**. Alternatively, it can be done in Xcode under **Signing&Capabilities** of the given target. 
```

Comment:
```suggestion
In case of getting errors about an existing provisioning profile's missing entitlements, you can add or update the entitlements by finding and editing the profile in **Apple Developer portal > Certificates, identifier & profiles > Identifiers**. Alternatively, it can be done in Xcode under **Signing & Capabilities** of the given target. 
```